src/common/types/store-data/unified-data-interface.ts

```diff
@@ -73,6 +73,15 @@ export type UnifiedResolution = {
     howToFixSummary: string;
 } & InstancePropertyBag;
 
+export type FormattedResolution = {
+    howToFix: string;
+    formatAsCode?: string[];
+};
+
+export type UnifiedFormattedResolution = {
+    howToFixFormat?: FormattedResolution;
```

Comment:
should this be named `howToFixFormatted` or `formattedHowToFix`??? `Format` implies a different meaning I guess